websites/H/HBO Max/presence.ts

```diff
@@ -126,14 +127,39 @@ async function fetchCover() {
 	return output;
 }
 
+async function fetchTitle() {
+	const accessToken = await fetchToken(),
+		{ routeKey, countryCode } = await fetchClientConfig(accessToken);
+
+	isFetching = true;
+
+	try {
+		const response = await fetch(
+			`https://comet${routeKey}.api.hbo.com/express-content/${
+				location.pathname.split("/")[2]
+			}?device-code=desktop&product-code=hboMax&api-version=v9.0&country-code=${countryCode}&language=en-us`,
+			{
+				headers: {
+					authorization: `Bearer ${accessToken}`,
+				},
+			}
+		).then(res => res.json());
+
+		titles = response[0].body.titles.short;
+	} catch {
+		return;
+	}
+
+	isFetching = false;
+}
```

Comment:
Can't you just edit `fetchCover()`? And you do realize `isFetching` wouldn't do anything here right

Reply:
Ok